packages/playground/windows/playground-win32/Playground-win32.vcxproj

```diff
@@ -9,10 +9,7 @@
     <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  <PropertyGroup>
-    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
-  </PropertyGroup>
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactPackageDirectories.props" />
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactDirectories.props" />
```

Comment:
Isn't this already included in `vnext\Directory.Build.props`?

I see this project lives outside `vnext`. Can you confirm it's explicitly included because of this?

Reply:
Playground (non-win32) was able to correctly build locally, meaning it was able to pick up the properties. It it wasn't included, it would fail pretty hard.